tests/benchmarks/timing_info/prj_userspace.conf

```diff
@@ -6,3 +6,4 @@ CONFIG_MAIN_STACK_SIZE=2048
 CONFIG_FORCE_NO_ASSERT=y
 CONFIG_APPLICATION_DEFINED_SYSCALL=y
 CONFIG_TEST_USERSPACE=y
+CONFIG_MP_NUM_CPUS=1
```

Comment:
FWIW: the reason it's done with only 1 cpu is because it's a benchmark and needs to run on a single processor for the measurements to have meaning.  Crashes are probably a bug and should be tracked (though if they're a bug in the test, we don't care because that's not how the test is intended to run, I guess).